### PR TITLE
Assert that valid fixtures contain no disables

### PR DESCRIPTION
**Pull request by edg2s**
Valid fixtures should contain only valid code. Adding an inline
disable should not be allowed.

In rare edge cases we need to trigger an error to assert that
another rule is valid, in which case allow an override.

### CODE REVIEW

File: test/fixtures/mediawiki/valid.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
 			This line asserts valid code for the no-child-content rule, but there's no way
 			to do that without running afoul of the no-v-html rule
 		-->
-		<!-- eslint-disable-next-line vue/no-v-html -->
+		<!-- eslint-disable-next-line vue/no-v-html !allowdisable -->
```
Comment (jdforrester) [inline test/fixtures/mediawiki/valid.vue:24]: Joy.

**Review by jdforrester — APPROVED** [review]
